pkgs/top-level/python-packages.nix

```
@@ -168,6 +168,10 @@ in {
 
   aioimaplib = callPackage ../development/python-modules/aioimaplib { };
 
+  aiolifx = callPackage ../development/python-modules/aiolifx { };
+
+  aiolifx_effects = callPackage ../development/python-modules/aiolifx_effects { };
```

```suggestion
  aiolifx-effects = callPackage ../development/python-modules/aiolifx-effects { };
```
See https://nixos.org/nixpkgs/manual/#contributing.